packaging/debian/copyright

```diff
@@ -0,0 +1,17 @@
+
+Copyright:
+
+    Copyright (c) 2017, OSG
```

Comment:
The OSG is not a legal entity.  Original copyright should go to the original author of the files.